README.md

```diff
@@ -94,114 +94,130 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
-
-## Load Plugins
-snap gets its power from the use of plugins. The [plugin catalog](#plugin-catalog) is a collection of all known plugins for snap.
-
-Next, open a new shell and lets load a few of the demo plugins.  You can do this via cURL, or `snapctl`, snap's CLI.
-
-Using cURL
-```sh
-$ cd $SNAP_PATH
-$ curl -X POST -F plugin=@plugin/snap-collector-mock1 http://localhost:8181/v1/plugins
-$ curl -X POST -F plugin=@plugin/snap-processor-passthru http://localhost:8181/v1/plugins
-$ curl -X POST -F plugin=@plugin/snap-publisher-file http://localhost:8181/v1/plugins
-```
-
-Or:
-
-Using `snapctl` (snap CLI)
+Every interaction with `snapd` can be done through the REST API. To continue using it, see the [API Documentation](docs/REST_API.md). We will continue on using `snapctl`:
 ```
-cd $SNAP_PATH
-$ <snapBinPath>/snapctl plugin load plugin/snap-collector-mock1
-$ <snapBinPath>/snapctl plugin load plugin/snap-processor-passthru
-$ <snapBinPath>/snapctl plugin load plugin/snap-publisher-file
+$ cd ~/snap/plugins/
+$ snapctl plugin load snap-collector-mock1
+$ snapctl plugin load snap-processor-passthru
+$ snapctl plugin load snap-publisher-file
 ```
 
-Let's look at what plugins we have loaded now:
-
+Let's look at what plugins you have loaded now:
 ```
-$ <snapBinPath>/snapctl plugin list
+$ snapctl plugin list
 NAME             VERSION         TYPE            SIGNED          STATUS          LOADED TIME
 mock             1               collector       false           loaded          Tue, 17 Nov 2015 14:08:17 PST
 passthru         1               processor       false           loaded          Tue, 17 Nov 2015 14:16:12 PST
 file             3               publisher       false           loaded          Tue, 17 Nov 2015 14:16:19 PST
 ```
-### Running Tasks
 
-Tasks can be in JSON or YAML format. Let's start one of the [example tasks](./examples/tasks/mock-file.yaml) from the `examples/tasks/` directory:
+You now have one of each plugin type loaded into the framework. To begin collecting data, you need to create a task.
+
+### Running Tasks
+Tasks are most often shared as a Task Manifest and is written in JSON or YAML format. Make a copy of [this example task](./examples/tasks/mock-file.yaml) from the `examples/tasks/` directory on your local system and then start the task:
 
 ```
-$ cd <snapDirectoryPath>
-$ <snapBinPath>/snapctl task create -t examples/tasks/mock-file.yaml
+$ cd ~/snap
+$ curl https://raw.githubusercontent.com/intelsdi-x/snap/master/examples/tasks/mock-file.yaml > mock-file.yaml
+$ snapctl task create -t mock-file.yaml
 Using task manifest to create task
 Task created
 ID: 8b9babad-b3bc-4a16-9e06-1f35664a7679
 Name: Task-8b9babad-b3bc-4a16-9e06-1f35664a7679
 State: Running
 ```
 
-From here, you should be able to do 2 things:
+This task generates mock data, "processes" it through a passthru mechanism and then publishes it to a file. Now with a running task, you should be able to do 2 things:
```

Comment:
Though the plugin is called passthru, if you're describing what it does it should probably be "pass-through" or whatever the full writing is...I'd also probably switch the existing 2 things with two things.